src/err-msg.js

```diff
@@ -0,0 +1,6 @@
+export function errMsg(errCode, msg) {
```

Comment:
I created a new file for this, so that extras can import this function into their build without incurring the byte cost of all the side effects within common.js (baseUrl check, etc)

Reply:
That sounds sensible.